### PR TITLE
Reorganize related articles into the `Client` category

### PR DESCRIPTION
**Pull request by TicClick**
part of #6080

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] `SKIP_OUTDATED_CHECK` for `wiki/Guides/How_to_Time_Songs/ru.md`

## Moves

- `Beatmap Editor` -> `Client/Beatmap editor`
- `osu! File Formats` -> `Client/File formats`
- `Installation` -> `Client/Installation`
- `Interface` -> `Client/Interface`
- `Chat Console` -> `Client/Interface/Chat console`
- `Glossary/Cookie` -> `Client/Interface/Cookie`
- `Glossary/Health bar` -> `Client/Interface/Health bar`
- `Visual Settings` -> `Client/Interface/Visual settings`
- `Shortcut key reference` -> `Client/Keyboard shortcuts`
- `Options` -> `Client/Options`
- `Glossary/Ignore list` -> `Client/Options/Ignore list`
- `osu! Program Files` -> `Client/Program files`
- `Universal offset` -> `Client/Options/Universal offset`

### CODE REVIEW

File: wiki/Client/en.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+stub: true  # this could make for a good introductory page
+tags:
+  game client
```
Comment (Walavouchey) [inline wiki/Client/en.md]:
```suggestion
  - game client
```

File: wiki/Client/en.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+stub: true  # this could make for a good introductory page
```
Comment (Walavouchey) [inline wiki/Client/en.md]: very sus

```suggestion
stub: true
```

File: wiki/Client/en.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+tags:
+  game client
+---
+
```
Comment (Walavouchey) [inline wiki/Client/en.md]:
```suggestion

<!-- TODO: this could make for a good introductory page -->

```

File: wiki/Client/en.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+Index page for articles about the osu! game client and its main sections.
+
+## Introduction
+
+- [Installation](/wiki/Client/Installation)
+- [Keyboard shortcuts](/wiki/Client/Keyboard_shortcuts)
+- [Interface](/wiki/Client/Interface)
+- [Options](/wiki/Client/Options)
+- [Beatmap editor](/wiki/Client/Beatmap_editor)
```
Comment (Walavouchey) [inline wiki/Client/en.md]:
```suggestion
- [Installation](Installation)
- [Keyboard shortcuts](Keyboard_shortcuts)
- [Interface](Interface)
- [Options](Options)
- [Beatmap editor](Beatmap_editor)
```

File: wiki/Client/en.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+## Technical details
+
+- [File formats](/wiki/Client/File_formats)
+- [Program files](/wiki/Client/Program_files)
```
Comment (Walavouchey) [inline wiki/Client/en.md]:
```suggestion
- [File formats](File_formats)
- [Program files](Program_files)
```

File: wiki/Client/Installation/zh-hk.md [hunk 6]
```diff
@@ -28,7 +30,9 @@ outdated: true
 - 享受 osu! 帶來的樂趣！
 
 ###### 添加譜面
-一個譜面是包含歌曲和播放所需要的文件。包括 .osz 文件。
+
+個譜面是包含歌曲和播放所需要的文件。包括 .osz 文件。
```
Comment (Walavouchey) [inline wiki/Client/Installation/zh-hk.md:34]: that's a chinese character and it means "one", put it back >:(

```suggestion
一個譜面是包含歌曲和播放所需要的文件。包括 .osz 文件。
```

**Review by Walavouchey — DISMISSED** [review]
Looked through at the moved files in a checkout of this branch and it seems okay at a quick glance

File: wiki/Client/Installation/zh-hk.md (hunk 6 above)
Comment (TicClick) [reply wiki/Client/Installation/zh-hk.md:34]: TIL

File: wiki/Client/en.md (hunk 2 above)
Comment (TicClick) [reply wiki/Client/en.md]: while this is valid, as front matter is YAML, I agree it's best read as a TODO

File: wiki/redirect.yaml [hunk 7]
```diff
@@ -1443,10 +1474,6 @@
 "tournaments/igts/2018": "Tournaments/GTS/IGTS_2018"
 "tournaments/igts/2019": "Tournaments/GTS/IGTS_2019"
 
-# Guide was replaced by a different article
-"how_to_run_osu!_on_your_mac_without_using_boot_camp":        "Installation/macOS"
-"guides/how_to_run_osu!_on_your_mac_without_using_boot_camp": "Installation/macOS"
-
```
Comment (cl8n) [inline wiki/redirect.yaml]: should be updated instead of deleted

**Review by cl8n — CHANGES_REQUESTED** [review]
I think it would be better to keep new entries to redirect.yaml chronological, it's easier to get a reference in the future for why they were added/changed
but if that's just how things are done now then nvm :runner:

File: wiki/redirect.yaml (hunk 7 above)
Comment (TicClick) [reply wiki/redirect.yaml]: fixed

**Comment by TicClick** [comment]
should be in a slightly better shape now

**Review by peppy — APPROVED** [review]

**Pull request merged**